Source/Rows/PickerInputRow.swift

```diff
@@ -28,19 +28,20 @@ import Foundation
 
 open class PickerInputCell<T: Equatable> : Cell<T>, CellType, UIPickerViewDataSource, UIPickerViewDelegate where T: Equatable, T: InputTypeInitiable {
     
-    public var picker: UIPickerView
+    lazy public var picker: UIPickerView = {
```

Comment:
I remember that we had issues with lazy variables using Cocoapods. Did you test that this works correctly?